src/utils/report-log.mapper.ts

```diff
@@ -8,27 +8,30 @@
 import { Log, Report, ReportSeverity } from '../types/report.type';
 import { REPORT_SEVERITY } from '../constants/report.constant';
 
-export const mapReportLog = (report: Report) => {
+export const mapReportLog = (report: Report, severities: String[]) => {
     const formattedLogs: Log[] = [];
-    formatReportLog(report, formattedLogs);
+    formatReportLog(report, severities, formattedLogs);
     return formattedLogs;
 };
 
-const formatReportLog = (report: Report, formattedLogs: Log[]) => {
+const formatReportLog = (report: Report, severities: String[], formattedLogs: Log[]) => {
     const highestSeverity = mapSeverity(report.severities ?? [REPORT_SEVERITY.UNKNOWN.name]);
-    // For now, we want to display only reports that are not containers in the "log" view
+    // We display a report line in the "log" view for both
+    // - a leaf (no sub-report)
+    // - and a container (have sub-reports), if its highest severity belongs to the severity filter (or unk)
     if (
-        report.subReports.length === 0 &&
-        highestSeverity.name !== REPORT_SEVERITY.UNKNOWN.name &&
-        report.parentId !== null
+        report.parentId &&
```

Comment:
I prefer to keep explicit check on null, in JS empty string is falsy for example so `if (report.parentId)` could mean a lot of different stuff

Reply:
I removed '!== null" on purpose to solve an issue ...

When we retrieve a report[] from the back, we build a meta-report called "Logs" having an undefined parent_id. Then we display a log "Logs" on the right side.

Now I set a null parent in the "Logs" report.

Reply:
oh ok, so maybe let it undefined and check report.parentId !== undefined ?

Reply:
yes I was hesitating between the 2 options. Let's use undefined.

Reply:
or "!= null" which works for both null and undefined